inc/settings.php

```diff
@@ -551,6 +556,7 @@ function siteorigin_north_font_settings( $settings ) {
 function siteorigin_north_settings_custom_css( $css ) {
 	// Custom CSS.
 	$css .= '/* style */
+	/**** /private/var/folders/qh/gg52m_yj7zv3bk6hb2x2j9w40000gn/T/JrS6rB/sass/style.css ***/
```

Comment:
I suspect this was added by mistake.

Reply:
Thanks, will remove shortly. The CSS builder adds those comments, they need to be manually removed.